system/keymaps/remote.xml

```diff
@@ -577,16 +577,28 @@
     <remote>
       <left>StepBack</left>
       <right>StepForward</right>
-      <up>ChannelUp</up>
-      <down>ChannelDown</down>
+      <up>Up</up>
+      <down>Down</down>
+      <skipplus>SkipNext</skipplus>
+      <skipminus>SkipPrevious</skipminus>
+      <pageup>ChannelUp</pageup>
+      <pagedown>ChannelDown</pagedown>
+      <channelplus>ChannelUp</channelplus>
+      <channelminus>ChannelDown</channelminus>
```

Comment:
we never use it, but they are part of the button translator. imo we should not map these in our default keymaps